unicodetools/data/ucd/dev/ScriptExtensions.txt

```diff
@@ -618,17 +618,17 @@ A830..A832    ; Deva Dogr Gujr Guru Khoj Knda Kthi Mahj Mlym Modi Nand Shrd Sind
 
 # ================================================
 
-# Script_Extensions=Beng Deva Dogr Gong Gonm Gran Gujr Guru Knda Mahj Mlym Nand Orya Sind Sinh Sylo Takr Taml Telu Tirh
+# Script_Extensions=Beng Deva Dogr Gong Gonm Gran Gujr Guru Knda Mahj Mlym Nand Onao Orya Sind Sinh Sylo Takr Taml Telu Tirh
```

Comment:
I am probably missing something, but L2/22-151R has it as `Olon`, not `Onao` — why is it different?

Reply:
@markusicu works in mysterious ways :-)

Reply:
I don't always follow the suggestions for script codes, just like we don't always follow suggested property values.
In this case, the approved code fits established patterns for existing codes.

Reply:
Another answer is that the authoritative list of script codes is at https://www.unicode.org/iso15924/iso15924-codes.html and the recent changes are easy to see at https://www.unicode.org/iso15924/codechanges.html

Reply:
OK. The actual code used doesn't matter to me, I just wanted to connect the dots on the difference. I'm satisfied with these explanations 😸